src/main/scala/org/broadinstitute/dsde/firecloud/model/OrchMethodRepository.scala

```diff
@@ -6,7 +6,7 @@ import org.broadinstitute.dsde.rawls.model.{AgoraMethod, DockstoreMethod, Method
 
 import scala.util.Try
 
-object MethodRepository {
```

Comment:
This seemed to conflict with a `MethodRepository` class included in rawls-model, which caused tons of issues with the implicits contained here.